Installer/JobsSetup/Product.wxs

```diff
@@ -72,6 +72,16 @@
 	                    Name="Version"
 	                    Type="raw" />
 	  </Property>
+
+	  <!-- powerJobs -->
+	  <Property Id="POWERJOBSINSTALLEDVERSION">
+		<RegistrySearch Id="PowerGateInstalledVersion"
+						Root="HKLM"
+						Key="SOFTWARE\coolOrange s.r.l.\powerJobs"
+						Name="Version"
+						Type="raw" />
+	  </Property>
+
 	  <Condition
 	    Message="This application requires powerVault v19 or later. Please install the latest version of powerVault and then run again this installer.">
 	    <![CDATA[Installed OR POWERVAULTINSTALLEDVERSION > "19"]]>
```

Comment:
In diesen Zuge kannst du gleich alle Versionen von v19 auf v22 ändern.
Aber Achtung für powerGate ist das neuste v21 glaube ich, wäre auf den Wiki gegenzuprüfen.

Reply:
done

Reply:
Ich seh die Änderungne für JobInstaller, aber für Client und Plugin installer nicht.
Kannst du das noch nachholen falls es noch nicht drinnen ist mit den neusten Versionen? Evtl Wiki auch nachnbessern falls notwendig.

Reply:
Ich werde dafür ein neues issue machen und noch diese Woche umsetzen, ok?